README.md

```diff
@@ -83,3 +83,22 @@ Due to [network limitation](https://docs.docker.com/docker-for-mac/networking/#k
 ```sh
 $ EXPOSE_HOST_PORTS=true make docker-testacc
 ```
+
+Releasing the Provider
+---------------------------
+
+* Create a draft of the [release](https://github.com/rancher/terraform-provider-rancher2/releases) and select create new tag for the version you are releasing
+* Create release notes by clicking `Generate release notes`
+* Copy the release notes to the CHANGELOG and update to the following format
+
+```
+# <tag version> (Month Day, Year)
+FEATURES:
+ENHANCEMENTS:
+BUG FIXES:
+```
+
+* Create a PR to update CHANGELOG
+* Copy the updated notes back to the draft release (DO NOT release with just the generated notes. Those are just a template to help you)
+* Publish the release (using the CL: git tag v0.0.0; git push -u origin v0.0.0)
```

Comment:
```suggestion
* Make sure the branch is up-to-date with the remote, in this example, the branch is master and the release tag is v1.24.0
git remote add upstream-release git@github.com:rancher/terraform-provider-rancher2.git
git checkout upstream-release/master
git push upstream-release v1.24.0
```
Based on https://github.com/rancher/rancher/blob/release/v2.6/docs/release.md#create-the-release-tag-and-push

I couldn't put the above code section in a code block but you get the idea.